### ISSUE TITLE
version 0.8.0 breaks import of `emojis` package, imported via ember-auto-import

### ISSUE THREAD

**Issue by NullVoxPopuli** [embroider-build/embroider#354]
Just upgraded to 0.8.0 to see if my previous test issue was resolved, new issue though:

```
ERROR Summary:

  - broccoliBuilderErrorStack: ModuleNotFoundError: Module not found: Error: Can't resolve './characters' in '/home/lprestonsegoiii/Development/NullVoxPopuli/emberclear/packages/frontend/node_modules/emojis'
    at factory.create (/home/lprestonsegoiii/Development/NullVoxPopuli/emberclear/packages/frontend/node_modules/webpack/lib/Compilation.js:821:10)
    at factory (/home/lprestonsegoiii/Development/NullVoxPopuli/emberclear/packages/frontend/node_modules/webpack/lib/NormalModuleFactory.js:397:22)
    at resolver (/home/lprestonsegoiii/Development/NullVoxPopuli/emberclear/packages/frontend/node_modules/webpack/lib/NormalModuleFactory.js:130:21)
    at asyncLib.parallel (/home/lprestonsegoiii/Development/NullVoxPopuli/emberclear/packages/frontend/node_modules/webpack/lib/NormalModuleFactory.js:224:22)
    at /home/lprestonsegoiii/Development/NullVoxPopuli/emberclear/packages/frontend/node_modules/neo-async/async.js:2830:7
    at /home/lprestonsegoiii/Development/NullVoxPopuli/emberclear/packages/frontend/node_modules/neo-async/async.js:6877:13
    at normalResolver.resolve (/home/lprestonsegoiii/Development/NullVoxPopuli/emberclear/packages/frontend/node_modules/webpack/lib/NormalModuleFactory.js:214:25)
    at doResolve (/home/lprestonsegoiii/Development/NullVoxPopuli/emberclear/packages/frontend/node_modules/enhanced-resolve/lib/Resolver.js:184:12)
    at hook.callAsync (/home/lprestonsegoiii/Development/NullVoxPopuli/emberclear/packages/frontend/node_modules/enhanced-resolve/lib/Resolver.js:238:5)
    at _fn0 (<anonymous>:15:1)
    at resolver.doResolve (/home/lprestonsegoiii/Development/NullVoxPopuli/emberclear/packages/frontend/node_modules/enhanced-resolve/lib/UnsafeCachePlugin.js:37:5)
    at hook.callAsync (/home/lprestonsegoiii/Development/NullVoxPopuli/emberclear/packages/frontend/node_modules/enhanced-resolve/lib/Resolver.js:238:5)
    at _fn0 (<anonymous>:15:1)
    at hook.callAsync (/home/lprestonsegoiii/Development/NullVoxPopuli/emberclear/packages/frontend/node_modules/enhanced-resolve/lib/Resolver.js:238:5)
    at _fn0 (<anonymous>:27:1)
    at resolver.doResolve (/home/lprestonsegoiii/Development/NullVoxPopuli/emberclear/packages/frontend/node_modules/enhanced-resolve/lib/DescriptionFilePlugin.js:42:38)
    at hook.callAsync (/home/lprestonsegoiii/Development/NullVoxPopuli/emberclear/packages/frontend/node_modules/enhanced-resolve/lib/Resolver.js:238:5)
    at _fn5 (<anonymous>:16:1)
    at hook.callAsync (/home/lprestonsegoiii/Development/NullVoxPopuli/emberclear/packages/frontend/node_modules/enhanced-resolve/lib/Resolver.js:238:5)
    at _fn0 (<anonymous>:27:1)
    at resolver.doResolve (/home/lprestonsegoiii/Development/NullVoxPopuli/emberclear/packages/frontend/node_modules/enhanced-resolve/lib/DescriptionFilePlugin.js:42:38)
    at hook.callAsync (/home/lprestonsegoiii/Development/NullVoxPopuli/emberclear/packages/frontend/node_modules/enhanced-resolve/lib/Resolver.js:238:5)
    at _fn1 (<anonymous>:16:1)
    at hook.callAsync (/home/lprestonsegoiii/Development/NullVoxPopuli/emberclear/packages/frontend/node_modules/enhanced-resolve/lib/Resolver.js:238:5)
    at _fn0 (<anonymous>:15:1)
    at fs.stat (/home/lprestonsegoiii/Development/NullVoxPopuli/emberclear/packages/frontend/node_modules/enhanced-resolve/lib/DirectoryExistsPlugin.js:22:13)
    at process.nextTick (/home/lprestonsegoiii/Development/NullVoxPopuli/emberclear/packages/frontend/node_modules/enhanced-resolve/lib/CachedInputFileSystem.js:73:15)
    at process._tickCallback (internal/process/next_tick.js:61:11)


```

repro: https://github.com/NullVoxPopuli/emberclear/pull/187

`characters` is a json file. does that matter?

**Comment by ef4** [comment]
I don't think this is a new bug in 0.8.0. More likely we fixed whatever earlier bug was concealing this one in your build.

I see that ember-auto-import relies on webpack's default [resolve.extensions](https://webpack.js.org/configuration/resolve/#resolveextensions) whereas embroider customizes it. When we customized it, we dropped json.

We should add it back in, because too many third-party libraries are going to expect this to work. This also means we should make it clear in the V2 spec that importing JSON is legal, and how that works.
